yarn-project/end-to-end/src/fixtures/l1_to_l2_messaging.ts

```diff
@@ -27,17 +27,18 @@ export async function sendL1ToL2Message(
   const version = await new RollupContract(ctx.l1Client, ctx.l1ContractAddresses.rollupAddress.toString()).getVersion();
 
   // We inject the message to Inbox
-  const txHash = await inbox.write.sendL2Message([
-    { actor: recipient.toString(), version: BigInt(version) },
-    content.toString(),
-    secretHash.toString(),
-  ]);
+  const txHash = await inbox.write.sendL2Message(
+    [{ actor: recipient.toString(), version: BigInt(version) }, content.toString(), secretHash.toString()],
+    {
+      gas: 1_000_000n,
+    },
+  );
   logger.info(`L1 to L2 message sent in tx ${txHash}`);
 
   // We check that the message was correctly injected by checking the emitted event
   const txReceipt = await ctx.l1Client.waitForTransactionReceipt({ hash: txHash });
```

Comment:
Unrelated to the code changes in this PR but `waitForTransactionReceipt` resolves receipts for reverted txs too. We don't properly check the status of the tx and act as if it was successful

Reply:
ah yeah i did actually change the code here and figured this out but added too much debugging stuff so I reverted it.
Will add a basic check for 'success' and throw if not